modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/addConnectedSiteButtonAdditionalProps.js

```diff
@@ -0,0 +1,39 @@
+export default function additionalProps({
+	currentURL,
+	eventName,
+	itemSelectorURL,
+	namespace,
+}) {
```

Comment:
We definitely need a function here... specially for ItemSelectorURLs, which are incredibly hard to generate.

Reply:
An example of this: https://github.com/liferay-frontend/liferay-portal/pull/2/files#diff-6a84f140f101bab7eb695d2c63da5052R73